app/components/tour/tour.js

```diff
@@ -25,9 +25,15 @@ const Tour = ({ onClose, ...props }) => {
         {
           content: (
             <div>
+              <p>Welcome to Hypergraph (Beta) 🥳</p>
               <p>
-                Welcome to Hypergraph (Beta) 🥳 We'll get you started with a
-                quick tour of the most important things.
+                The beta has some rough edges, so please be patient and kind to
+                it (especially when downloading files). Maybe take a break while
+                you're waiting.
```

Comment:
```suggestion
                it (especially when downloading files).
```

We don't have a problem with it downloading slowly